app/containers/InterventionDetailsPage/containers/ParticipantInviter/InviteParticipantsModalContent.tsx

```diff
@@ -188,6 +191,10 @@ export const InviteParticipantsModalContent: FC<Props> = ({
     organizationLoading ||
     (isReportingIntervention && organizationId !== organization?.id);
 
+  const filteredSessionOptions = sessionOptions.filter(
+    (session) => session.type !== 'Session::Sms',
```

Comment:
If you really would need to filter by session type you should compare to `SessionTypes.SMS_SESSION` , not a plain string